DESCRIPTION

```diff
@@ -86,7 +86,6 @@ Collate:
     'simulation-run-options.R'
     'simulation-settings.R'
     'simulation.R'
-    'simulationResultsToDataFrame.R'
```

Comment:
Why is it not exported any more?

Reply:
It is exported. The field you are looking at is `Collate`, which is just showing to you that the `.R` file was removed.

Look at `NAMESPACE` file instead, and you will see that this function is exported.

Reply:
Ah ok then.